lib/box/index.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+
+var box = angular.module('guBox', []);
+
+box.directive('guBox', function boxDirective(){
+  return {
+    restrict: 'AE',
```

Comment:
I'm not sure I'd have to check the docs. Could you cheekily check if `el.tagName == 'GU-BOX'`?